### ISSUE TITLE
runtime: unknown pc (random crash) - Geth/v1.9.14, Windows

### ISSUE THREAD

**Issue by remyroy** [ethereum/go-ethereum#21093]
#### System information

Geth version: `geth version`

```
C:\Users\Rémy Roy\ethereum>geth version
Geth
Version: 1.9.14-stable
Git Commit: 6d74d1e5f762e06a6a739a42261886510f842778
Git Commit Date: 20200513
Architecture: amd64
Protocol Versions: [65 64 63]
Go Version: go1.14.2
Operating System: windows
GOPATH=
GOROOT=C:\go
```

OS & Version: Windows 10

#### Expected behaviour
No random crash during normal usage.

#### Actual behaviour
Crashed randomdly after running for a few hours

#### Steps to reproduce the behaviour
Unclear to be honest.

- Start geth on windows using those parameters:

```
geth.exe --txlookuplimit 100 --nousb --ipcdisable --verbosity 1 console
```

- Let it run for a few hours.
- Wait random event.
- Notice the crash.

#### Backtrace

See https://gist.github.com/remyroy/20e5f08ca9bd84dca90b21ac87f35fd9 for the full backtrace.

**Comment by karalabe** [comment]
We've no clue :)

The error is a Windows exception, which seems to indicate that the program accessed something it shouldn't be able to. It might be a memory corruption, faulty memory and there's a slight chance to be in Geth's crypto. Given that noone else reported it, it might be worth to do a mem check on your machine to see if everything's ok. Also please check with the latest release, we've upgraded to Go 1.14.4, which may or may not have fixed something.

**Comment by MariusVanDerWijden** [comment]
@remyroy This issue is getting kinda stale, have you seen this problem with a current version too?

**Comment by remyroy** [comment]
> @remyroy This issue is getting kinda stale, have you seen this problem with a current version too?

No. I've been running various Geth version on the same machine in a Linux VM without any issue. This might have been fixed or this might be a Windows related issue. In any case, it might not be worth spending more time on it.
